### PR TITLE
Fix unable to place large schematics

### PR DESCRIPTION
**Pull request by zxtej**
attempt 2

### CODE REVIEW

File: core/src/mindustry/Vars.java [hunk 1]
```diff
@@ -246,6 +246,8 @@
 
     public static Player player;
     public static boolean drawCursors, wasDrawingCursors; // Client debug magic
+    public static IntMap<Object> processorConfigMap = new IntMap<>();
```
Comment (buthed010203) [inline core/src/mindustry/Vars.java]: Use ClientVars.java instead

File: core/src/mindustry/client/ClientLogic.kt [hunk 2]
```diff
@@ -28,6 +28,10 @@ class ClientLogic {
         Events.on(EventType.WorldLoadEvent::class.java) { // Run when the world finishes loading (also when the main menu loads and on syncs)
             Core.app.post { syncing = false } // Run this next frame so that it can be used elsewhere safely
             if (!syncing) Vars.player.persistPlans.clear() // FINISHME: Why is this even in the player class? It creates a queue for each player even tho only one is used...
+            if (!syncing){
+                Vars.player.persistPlans.clear()
```
Comment (buthed010203) [inline core/src/mindustry/client/ClientLogic.kt:35]: Duplicated code

File: core/src/mindustry/core/Logic.java [hunk 3]
```diff
@@ -56,6 +64,8 @@ public Logic(){
             }
         });
 
+        Events.on(ClientLoadEvent.class, e -> Core.app.post(() -> setProcessorBypassHack(Core.settings.getBool("prochack", false))));
```
Comment (buthed010203) [inline core/src/mindustry/core/Logic.java]: ClientLogic.java exists

File: core/src/mindustry/client/ClientLogic.kt (hunk 2 above)
Comment (zxtej) [reply core/src/mindustry/client/ClientLogic.kt:35]: Oops, didn't see that

**Comment by buthed010203** [comment]
Did this slightly differently

**Pull request closed without merging**